Cargo.toml

```
@@ -7,32 +7,22 @@ license = "GPL-3.0-or-later"
 readme = "README.md"
 description = "Quadrature integration"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [lib]
 name = "quad"
-# "cdylib" is necessary to produce a shared library for Python to import from.
-#
-# Downstream Rust code (including code in `bin/`, `examples/`, and `tests/`) will not be able
-# to `use python_bidings;` unless the "rlib" or "lib" crate type is also included, e.g.:
-# crate-type = ["cdylib", "rlib"]
 crate-type = ["cdylib"]
```

bench seems to work with "["cdylib", "rlib"]" crate-type

---

@AleCandido

---

```suggestion
crate-type = ["cdylib", "rlib"]
```

Just to show you how a suggestion looks like :)